workshop/content/30-python/70-advanced-topics/200-pipelines/5000-test-actions.md

```diff
@@ -136,7 +136,7 @@ With a slight modification to `cdk_workshop/pipeline_stage.py` we can expose the
 from aws_cdk import (
     core
 )
-from pypipworkshop_stack import PypipworkshopStack
+from cdk_workshop_stack import CdkWorkshopStack
```

Comment:
```suggestion
from .cdk_workshop_stack import CdkWorkshopStack
```

Reply:
added the fix here [b3d4f07](https://github.com/aws-samples/aws-cdk-intro-workshop/pull/320/commits/b3d4f073ff6fa29e8da4f583306ee65c6ccae9bf)